common/libs/keystore.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import { randomBytes, createCipheriv } from 'crypto';
+import { privateToPublic, publicToAddress, sha3 } from 'ethereumjs-util';
+import scrypt from 'scryptsy';
+import uuid from 'uuid';
+import { makeBlob } from 'libs/globalFuncs';
+
+export const scryptSettings = {
+  n: 1024
+};
+
+export const kdf = 'scrypt';
+
+export function genNewPkey(): Buffer {
+  return randomBytes(32);
+}
+
+export function pkeyToAddress(pkey: Buffer): string {
+  return publicToAddress(privateToPublic(pkey)).toString('hex');
+}
+
+export function pkeyToKeystore(pkey: Buffer, password: string) {
+  const salt = randomBytes(32);
+  const iv = randomBytes(16);
```

Comment:
It may be valuable to leave a comment explaining some of these values / why you landed on them.